entities/license/offering.json

```diff
@@ -89,7 +93,7 @@
         },
         "description": "List of bundled offering uids"
       },
-      "related_offerings":{
+      "pay_per_staff":{
         "type":"array",
```

Comment:
Why is this a list? Its a reference to a single offering with all the prices right?

Reply:
Because you might want to see all the bundles you have for a partner, on packages/apps, or created in the last month.